packages/mobile/patches/react-native-tiktok+1.1.1.patch

```diff
@@ -0,0 +1,69 @@
+diff --git a/node_modules/react-native-tiktok/ios/Podfile b/node_modules/react-native-tiktok/ios/Podfile
+index e0be15c..050d9a4 100644
+--- a/node_modules/react-native-tiktok/ios/Podfile
++++ b/node_modules/react-native-tiktok/ios/Podfile
+@@ -4,7 +4,7 @@
+ target 'Tiktok' do
+   # Comment the next line if you don't want to use dynamic frameworks
+   use_frameworks!
+-  pod 'TikTokOpenSDK', '~> 5.0.0'
++  pod 'TikTokOpenSDK', '5.0.0'
+ 
+   # Pods for Tiktok
+ 
+diff --git a/node_modules/react-native-tiktok/ios/Podfile.lock b/node_modules/react-native-tiktok/ios/Podfile.lock
+index be8a751..c55f3e8 100644
+--- a/node_modules/react-native-tiktok/ios/Podfile.lock
++++ b/node_modules/react-native-tiktok/ios/Podfile.lock
+@@ -2,7 +2,7 @@ PODS:
+   - TikTokOpenSDK (5.0.0)
+ 
+ DEPENDENCIES:
+-  - TikTokOpenSDK (~> 5.0.0)
++  - TikTokOpenSDK (= 5.0.0)
+ 
+ SPEC REPOS:
+   trunk:
+@@ -11,6 +11,6 @@ SPEC REPOS:
+ SPEC CHECKSUMS:
+   TikTokOpenSDK: a63564ea4dc1fabc56352bf7aa9ddc386cdb63b6
+ 
+-PODFILE CHECKSUM: 8d42bd334b852726cbc4757006ffc25b49f26532
++PODFILE CHECKSUM: cfad65ff3c8a124df8f8bf88a97e46e7de98bd5d
+ 
+-COCOAPODS: 1.10.1
++COCOAPODS: 1.11.3
+diff --git a/node_modules/react-native-tiktok/ios/Pods/Manifest.lock b/node_modules/react-native-tiktok/ios/Pods/Manifest.lock
+index be8a751..c55f3e8 100644
+--- a/node_modules/react-native-tiktok/ios/Pods/Manifest.lock
++++ b/node_modules/react-native-tiktok/ios/Pods/Manifest.lock
+@@ -2,7 +2,7 @@ PODS:
+   - TikTokOpenSDK (5.0.0)
+ 
+ DEPENDENCIES:
+-  - TikTokOpenSDK (~> 5.0.0)
++  - TikTokOpenSDK (= 5.0.0)
+ 
+ SPEC REPOS:
+   trunk:
+@@ -11,6 +11,6 @@ SPEC REPOS:
+ SPEC CHECKSUMS:
+   TikTokOpenSDK: a63564ea4dc1fabc56352bf7aa9ddc386cdb63b6
+ 
+-PODFILE CHECKSUM: 8d42bd334b852726cbc4757006ffc25b49f26532
++PODFILE CHECKSUM: cfad65ff3c8a124df8f8bf88a97e46e7de98bd5d
+ 
+-COCOAPODS: 1.10.1
++COCOAPODS: 1.11.3
+
```

Comment:
I think this line is an extra whitespace causing the patch to fail to apply. Removing it fixed things for me locally!

Reply:
ahh thanks!